src/main/java/org/terasology/blockdetector/systems/BlockDetectorSystemImpl.java

```diff
@@ -104,33 +103,26 @@
     private Set<Vector3i> detectedBlocks = new HashSet<>();
 
     /**
-    * This set the local player for the tests
+    * This sets the local player for the tests
     * The dummy local player will make the system work and does not throw a null exception
-    * @param set the local player that going to be set through the test method
     */
     public void setLocalPlayer(LocalPlayer set) {
         this.localPlayer = set;
     }
 
-    /**
-     * This provides the value for the test
```

Comment:
I'm fine with you removing it, but here is how i would have personally done it if I absolutely had to do it:
```
/**
* Returns the time since last update call.
* @return A float representing the time since last update call.
*/
```
See? The description is way more clear.
That being said, as you can see, it's not great, because it adds no real information... so I'm glad you removed this.